src/plugins/plugin-grpc.ts

```diff
@@ -14,38 +14,108 @@
  * limitations under the License.
  */
 
-'use strict';
+import {EventEmitter} from 'events';
+import * as grpcModule from 'grpc';  // for types only.
+import {Client, MethodDefinition, ServerReadableStream, ServerUnaryCall, StatusObject} from 'grpc';
+import * as shimmer from 'shimmer';
 
-var shimmer = require('shimmer');
-var findIndex = require('lodash.findindex');
+import {Plugin, RootSpan, RootSpanOptions, Span, TraceAgent} from '../plugin-types';
 
-var SKIP_FRAMES = 1;
+// Re-definition of Metadata with private fields
```

Comment:
Is there a way to get gRPC to ship a second module with just the types that could be used a devDependency here?

Reply:
I read code wrong.